scipy/stats/_multivariate.py

```diff
@@ -155,9 +155,12 @@ def __init__(self, M, cond=None, rcond=None, lower=True,
         # Compute the symmetric eigendecomposition.
         # Note that eigh takes care of array conversion, chkfinite,
         # and assertion that the matrix is square.
+        M = np.asarray(M)
         s, u = scipy.linalg.eigh(M, lower=lower, check_finite=check_finite)
 
         eps = _eigvalsh_to_eps(s, cond, rcond)
+        if not np.allclose(M, M.T, atol=1e-10):
```

Comment:
This test should move before `M` is used in the symmetric eigendecomposition.

I recommend using `scipy.linalg.issymmetric(M)` instead. Where did your selection of `atol` come from?

Reply:
Thanks! I changed it. My choice of `atol` came from the default value of `atol` in `sklearn.utils.validation.check_symmetric`.

https://github.com/scikit-learn/scikit-learn/blob/7e1e6d09bcc2eaeba98f7e737aac2ac782f0e5f1/sklearn/utils/validation.py#L1092